tests/apps_test.go

```diff
@@ -36,7 +36,7 @@ var _ = Describe("Apps", func() {
 			sess, err := start("deis logs -a %s", appName)
 			Expect(err).To(BeNil())
 			Eventually(sess).Should(Exit(1))
-			Eventually(sess.Err).Should(Say("Not found."))
+			Eventually(sess.Err).Should(Say(`Error: There are currently no log messages. Please check the following things:`))
```

Comment:
Oh I just didn't know what they did for us, that's all. Not holding this up. Just trying to learn.